webapp/src/js/services/tour.js

```diff
@@ -588,7 +590,16 @@ angular.module('inboxServices').service('Tour',
             });
           } else {
             // navigate to the correct page
-            $state.go(route, { tour: name });
+            if (route) {
+              $state.go(route, { tour: name });
+            } else {
+              var message = 'Attempt to navigate to an undefined state';  
```

Comment:
Gather more information here to help with debugging next time, for example, mention that you're in the TourController start function, and the name of the tour you were trying to load.